### PR TITLE
Some docs updates for the 0.2.0 release

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@JStickler, PTAL

File: servicemesh-install/topics/install-overview.adoc [hunk 1]
```diff
@@ -23,11 +23,13 @@ The operator creates an Ansible job that runs an Ansible playbook that performs
 ** istio-telemetry
 * Installs Elasticsearch
 * Installs Grafana
-* Installs the following Jaeger components (if configured in the custom resource definition):
+* Installs the following Jaeger components:
```
[the remark this answers is not in the document]
Comment (JStickler) [reply servicemesh-install/topics/install-overview.adoc]: Ah, I didn't understand that.  Thanks!

File: servicemesh-install/topics/install-overview.adoc [hunk 2]
```diff
@@ -23,11 +23,13 @@ The operator creates an Ansible job that runs an Ansible playbook that performs
...
 ** jaeger-collector
 ** jaeger-query
 * Installs Prometheus
+* Installs the following Kiali components (if configured in the custom resource definition):
+** kiali
 * Performs the following launcher configuration tasks (if configured in the custom resource definition):
 ** Creates a `devex` project and installs the Fabric8 launcher into that project.
-** Adds the cluster admin role to the OpenShift user specified in the launcher parameters in the custom resource file.
\ No newline at end of file
+** Adds the cluster admin role to the OpenShift user specified in the launcher parameters in the custom resource file.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install-overview.adoc]: s/OpenShift/{product-title} (multiple places, multiple files)

File: servicemesh-install/topics/install-prerequisites.adoc [hunk 3]
```diff
@@ -104,7 +106,36 @@ If deployment_type=openshift the default is `openshift-istio-tech-preview/`.
...
+|===
+
+.Kiali parameters
+|===
+|Parameter |Values |Description  |Default
+
+|`prefix`
+|Any valid image repo.
+|Which prefix to apply to the Kiali image name used in docker pull.
+|If deployment_type=origin the default is `kiali/`.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install-prerequisites.adoc]: s/deployment_type=origin/\`deployment_type=origin\` (two places)

File: servicemesh-install/topics/install-prerequisites.adoc [hunk 4]
```diff
@@ -104,7 +106,36 @@ If deployment_type=openshift the default is `openshift-istio-tech-preview/`.
...
+
+If deployment_type=openshift the default is `openshift-istio-tech-preview/`.
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install-prerequisites.adoc]: s/deployment_type=openshift/\`deployment_type=openshift\`,  (two places)

File: servicemesh-install/topics/install-prerequisites.adoc [hunk 5]
```diff
@@ -104,7 +106,36 @@ If deployment_type=openshift the default is `openshift-istio-tech-preview/`.
...
+
+|`version`
+|Any valid Docker tag.
+|Which Docker tag to use with the Kiali image.
+|If deployment_type=origin then `v0.7.2`.
+
+If deployment_type=openshift then `0.7.2`.
+
+|`username`
+|The username used to access the Kiali console. Note that this is not related to any account on OpenShift
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install-prerequisites.adoc]: s/username/user name

File: servicemesh-install/topics/install.adoc [hunk 6]
```diff
@@ -10,12 +10,18 @@ The {ProductShortName} installation process introduces a Kubernetes operator to
 == Installing the operator
 The following steps will install the {ProductShortName} operator into an existing {product-title} 3.10 installation; they can be executed from any host with access to the cluster.  Please ensure you are logged in as a cluster admin before executing the following commands.
 
-You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.1.0-ocp-3.1.0-istio-1.0.0/istio[operator templates on GitHub].
+You can find the https://github.com/Maistra/openshift-ansible/tree/maistra-0.2.0-ocp-3.1.0-istio-1.0.2/istio[operator templates on GitHub].
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install.adoc]: @vikram-redhat, are you ok with this GitHub reference?

File: servicemesh-install/topics/install.adoc [hunk 7]
```diff
@@ -10,12 +10,18 @@ The {ProductShortName} installation process introduces a Kubernetes operator to
...
 
 ```
 $ oc new-project istio-operator
 $ oc new-app -f istio_product_operator_template.yaml --param=OPENSHIFT_ISTIO_MASTER_PUBLIC_URL=<master public url>
 ```
+[NOTE]
+.OpenShift Master Public URL
+=====================
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install.adoc]: Four "=" signs

Remove the note label.

Rewrite to not use "should."

s/your OpenShift Console, this/{product-title}. This

**Review by kalexand-rh — COMMENTED** [review]
Couple of style picks. Unless you're referring to a specific OpenShift element, you can't just say "OpenShift." You need to use the {product-title} variable to apply the correct OpenShift product name for the collection.

File: servicemesh-install/topics/install-overview.adoc (hunk 2 above)
Comment (knrc) [reply servicemesh-install/topics/install-overview.adoc]: This should now be done

File: servicemesh-install/topics/install-prerequisites.adoc (hunk 3 above)
Comment (knrc) [reply servicemesh-install/topics/install-prerequisites.adoc]: This should now be done

File: servicemesh-install/topics/install-prerequisites.adoc (hunk 4 above)
Comment (knrc) [reply servicemesh-install/topics/install-prerequisites.adoc]: This should now be done

File: servicemesh-install/topics/install-prerequisites.adoc (hunk 5 above)
Comment (knrc) [reply servicemesh-install/topics/install-prerequisites.adoc]: This should now be done

File: servicemesh-install/topics/install.adoc (hunk 7 above)
Comment (knrc) [reply servicemesh-install/topics/install.adoc]: This should now be done

File: servicemesh-install/topics/install-prerequisites.adoc [hunk 8]
```diff
@@ -236,22 +267,6 @@ $ oc ex config patch master-config.yaml.prepatch -p "$(cat master-config.patch)"
 $ master-restart api
 $ master-restart controllers
 ```
-+
-. Then you must modify each individual deployment that you want to monitor as part of your service mesh to enable automatic sidecar injection. Each deployment where you want to enable automatic injection needs to contain the `sidecar.istio.io/inject: "true":` annotation.
```
Comment (brian-avery) [inline servicemesh-install/topics/install-prerequisites.adoc]: I don't necessarily think this belongs here, but this is the only place on the OSSM docs that mentions the annotation. I don't think we can entirely remove this.

**Review by brian-avery — CHANGES_REQUESTED** [review]

File: servicemesh-install/topics/install-prerequisites.adoc (hunk 8 above)
Comment (knrc) [reply servicemesh-install/topics/install-prerequisites.adoc]: This is more to do with how to deploy applications and take advantage of the automatic injection so has nothing to do with installation.  It definitely shouldn't be in this section, perhaps we need to introduce a new one covering how to develop applications using automatic injection?

Comment (brian-avery) [reply servicemesh-install/topics/install-prerequisites.adoc]: I agree. A separate section would be great. Just want to make sure this makes it into the doc.

**Review by brian-avery — APPROVED** [review]

**Comment by kalexand-rh** [comment]
@JStickler, @knrc, does this content still need to be merged?

**Comment by knrc** [comment]
@knrc Yes, I made all the requested changes and since then it has been sitting.  We are releasing 0.3.0 next week so there will be another PR issued later this week to update to that release.

**Comment by knrc** [comment]
@kalexand-rh There are now quite a few clashes so it looks as if I'll need to start afresh, rebase is throwing up too many differences.  I'll get this done later today.

**Comment by kalexand-rh** [comment]
@knrc, fair enough. Tag me in the new PR.

**Comment by knrc** [comment]
@kalexand-rh I'll still use this PR, GitHub will handle it automatically
when I push the updated branch.

File: servicemesh-install/topics/install.adoc (hunk 6 above)
Comment (vikram-redhat) [reply servicemesh-install/topics/install.adoc]: Hi Kevin - @knrc - is this the upstream for ServiceMesh?

Comment (knrc) [reply servicemesh-install/topics/install.adoc]: @vikram-redhat Yes it is.

Comment (vikram-redhat) [reply servicemesh-install/topics/install.adoc]: @knrc - is there a RH supported link instead? If not, is it confirmed within these docs that Service Mesh is not supported by RH and that this is dev preview only? If yes, we can approve this link.

Comment (knrc) [reply servicemesh-install/topics/install.adoc]: @vikram-redhat There is no product/supported link as of yet and this release is described as a tech preview within the docs and the images.

Comment (vikram-redhat) [reply servicemesh-install/topics/install.adoc]: Thanks Kevin (@knrc)! 

@kalexand-rh - yes we can accept this link.

**Comment by vikram-redhat** [comment]
@knrc looks like the rebase is required due to the changes introduced by this PR: https://github.com/openshift/openshift-docs/pull/12198

File: servicemesh-install/topics/install.adoc (hunk 6 above)
Comment (kalexand-rh) [reply servicemesh-install/topics/install.adoc]: Thanks Vikram! When Kevin clears the merge errors, I'll merge this PR. :)

**Comment by kalexand-rh** [comment]
@vikram-redhat, Kevin said that he'd fix up the PR soon.

**Comment by knrc** [comment]
@kalexand-rh @vikram-redhat I've been through all the changes and most are already applied to the docs, I've updated this PR to cover the remaining changes.

**Comment by kalexand-rh** [comment]
@knrc, there's a build error: ERROR: servicemesh-install.adoc: line 75: include file not found: /home/travis/build/openshift/openshift-docs/drupal-build/openshift-enterprise/servicemesh-install/topics/revision-info.adoc

Will you PTAL?

**Comment by knrc** [comment]
@kalexand-rh I can find no evidence that file was ever committed to the repository, I'm guessing someone forgot to add it.

**Comment by knrc** [comment]
@JStickler Any ideas?

**Comment by JStickler** [comment]
@knrc The servicemesh-install.adoc file does have a commented out reference to the servicemesh-install.adoc that was there in the templates.  Since I'm not using the revisions file, I never checked it in.  But that hasn't thrown any errors until now.  The only thing I can think of is if your text editor is configured to remove blank spaces at the end of files?  Or if the rebase somehow mangled that include being commented out?

**Comment by knrc** [comment]
@JStickler I gave up on the rebase in the end, too many differences.  I went through each change one by one and reapplied them to the appropriate location in the docs.  This file is not one I modified hence why I'm asking for help :)

**Comment by knrc** [comment]
@JStickler @kalexand-rh @vikram-redhat Looking at the file the contents have not changed in over a month, and still appear to be commented out, so is this an error with the tooling?

**Comment by vikram-redhat** [comment]
@knrc looking..

**Comment by knrc** [comment]
@vikram-redhat done

**Comment by vikram-redhat** [comment]
@knrc - thanks and the build passes. Cool. I will merge as this was already peer reviewed.

**Pull request merged**